Added info on supported Ubuntu versions

magnusoverli opened the pull request:
Adding a note on the latest docker-supported version of Ubuntu.

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9940"><img src="https://gitpod.io/api/apps/github/pbs/github.com/magnusoverli/home-assistant.io.git/b4799e0cb218952a64485ee63c0a222f7373b090.svg" /></a>

magnusoverli committed:
Added info on supported Ubuntu versions

Adding a note on the latest docker-supported version of Ubuntu.

frenck commented:
hmm... we'd have to keep it up to date, that is not really my favorite thing to add to be honest.

magnusoverli committed:
Added info on supported versions of Ubuntu

Added a note to remind users that docker needs to be supported by their version of Ubuntu, and where to check if it is.

frenck commented on source/hassio/installation.markdown:
```diff
@@ -136,6 +136,8 @@ You also need to have Docker-CE installed. There are well-documented procedures
   Some distributions, like Ubuntu, have a `docker.io` package available. Using that packages will cause issues!
   Be sure to install the official Docker-CE from the above listed URL.
+  
+  Docker is not always ready with a release when a new Ubuntu version is out. Check if your version of Ubuntu is supported by docker [here](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
```
```suggestion
Docker might not be supporting your operating system of choice. Check the [Docker documentation](https://docs.docker.com/install/#supported-platforms) and make sure your system is supported.
```

frenck replied:
Added suggestion to even remove Ubuntu out of it.

magnusoverli replied:
Isn’t it a bit awkward to remove Ubuntu from the description and still link to the Ubuntu specific documentation?

frenck replied:
Agreed, adjusted.

frenck commented:
Merging this one in. Thanks, @magnusoverli
